src/monster.h

```diff
@@ -323,6 +323,12 @@ class monster : public Creature, public JsonSerializer, public JsonDeserializer
         /** Returns vector containing all body parts this monster has. That is, { bp_torso } */
         std::vector<body_part> get_all_body_parts( bool main = false ) const override;
 
+        /*
+        * Decides what to do in case the monster is idle.
+        * If more idle actions are added change if loop for switch case.
```

Comment:
Not a switch, but rather something like monster attack - a json-coded function wrapper.